tutorials/napps/development_environment_setup.rst

```diff
@@ -38,16 +38,67 @@ Installing required dependencies
 ********************************
 
 In order to start using and coding with Kytos, you need a few required
-dependencies. One of them is Python 3.6 or greater.
+dependencies. One of them is Python 3.9.x
 
+
+Installing Python 3.9.x
+=====================
+
+Install the dependencies necessary to build Python:
+
+.. code-block:: console
+
+ sudo apt update
+ sudo apt install build-essential zlib1g-dev libncurses5-dev libgdbm-dev libnss3-dev libssl-dev libreadline-dev libffi-dev libsqlite3-dev wget libbz2-dev
+
+Download the latest release’s source code from the Python download page with wget :
+
+.. code-block:: console
+ 
+ wget https://www.python.org/ftp/python/3.9.1/Python-3.9.1.tgz
+
+Once the download is complete, extract the gzipped archive :
+
+.. code-block:: console
+
+ tar -xf Python-3.9.1.tgz
+
+Switch to the Python source directory and run the configure script, which performs a number of checks to make sure all of the dependencies on your system are present:
+
+.. code-block:: console
+
+ cd Python-3.9.1
+ ./configure --enable-optimizations
+
+Start the Python 3.9 build process:
+
+.. code-block:: console
+
+ make -j 12
+
+For faster build time, modify the -j to correspond to the number of cores in your processor. You can find the number by typing nproc.
+
+When the build process is complete, install the Python binaries by typing:
+
+.. code-block:: console
+
+ sudo make altinstall
+
+That’s it. Python 3.9 has been installed and ready to be used. To verify it, type:
+
+.. code-block:: console
+
+ python3.9 --version
+ 
 Required packages
 =================
 
 The required Ubuntu packages can be installed by:
 
 .. code-block:: console
 
-  $ apt install git libpython3-dev python3 python3-venv
+  $ apt install git libpython3.9-dev python3.9 python3.9-venv
```

Comment:
@usmanazFIU did you confirm if these python3.9 related packages ship natively with Ubuntu 20.04?

Reply:
they come seperately

Reply:
Right @usmanazFIU, so maybe we should add more information here not to confuse users. Could you complement the info on which ppa or something else that you need to have these? And then I think if we can make it clear that either the user add this ppa or repo, or either build Python from source as you explained in prior paragraph.